controller-client/proto/Controller.proto

```diff
@@ -1,11 +1,17 @@
 /**
- * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * Copyright Pravega Authors.
```

Comment:
I guess we need to replace this everywhere?

Reply:
I have created an issue https://github.com/pravega/pravega-client-rust/issues/272 to fix this seperately.